pages/common/gource.md

```diff
@@ -0,0 +1,31 @@
+# gource
+
+> Visualize the history of Git, SVN, Mercurial and Bazaar repositories.
+
+- Run gource in a directory:
```

Comment:
What happens if the directory isn't the root of the repository? Does the graph only show the portion of the project that's inside the given directory?